### ISSUE TITLE
Extend whiteboard to enable multiple users at the same time

### ISSUE THREAD

**Issue by bigbluebutton-issue-import** [bigbluebutton/bigbluebutton#2055]
Originally reported on Google Code with ID 1302

```
For certain applications, such as one-to-one tutoring, it would be useful for the teacher
and student to simultaneous interact with the whiteboard.

See

  https://groups.google.com/group/bigbluebutton-users/browse_thread/thread/c475eae54f9fa820#
```

Reported by `ffdixon` on 2012-08-29 22:09:30

**Comment by mrssantiago** [comment]
I am also  interested  in this  feature.  We oversee online schools that are 100% online and where students and teachers engage int he learning process from home.  During live sessions with teacher support of learning, it is necessary to have small groups or classes of students in the session with the teacher and the teacher needs a means by which she can assess student learning in real time and having to do it one at a time makes inefficient use of the time they have available.

Please let me know how I can find out about the progress with this feature.  Thank you in advance!

**Comment by ffdixon** [comment]
This feature is not tagged for any specific release, and it's not planned for the next release (BigBlueButton 1.1), see

    https://groups.google.com/d/msg/bigbluebutton-dev/2OLdUbi2K_w/1pzOZoppAwAJ

You'll know were getting closer to implementing this feature when we tag it for a release.

**Comment by ffdixon** [comment]
Implemented in BigBlueButton 2.0-dev!

![image](https://user-images.githubusercontent.com/290351/29470306-d94ea894-8419-11e7-9e7a-48d09342cca4.png)
